pull in in.h as it sets up some proto defines that iptables relies on

--- a/include/xtables.h.in
+++ b/include/xtables.h.in
@@ -5,6 +5,7 @@
 #include <sys/types.h>
 #include <stdbool.h>
 #include <net/if.h>
+#include <netinet/in.h>
 #include <linux/types.h>
 #include <linux/netfilter/x_tables.h>
 
